ingestion_server/ingestion_server/slack.py

```diff
@@ -3,30 +3,48 @@
 
 import requests
 from decouple import config
+from enum import Enum
 
 
 log = logging.getLogger(__name__)
 SLACK_WEBHOOK = "SLACK_WEBHOOK"
+LOG_LEVEL = "SLACK_LOG_LEVEL"
 
+class Level(Enum):
+    VERBOSE = 0
+    INFO = 1
+    ERROR = 2
 
-def message(text: str, summary: str = None) -> None:
+
+def _message(text: str, summary: str = None, level: Level = Level.INFO) -> None:
     """
     Send a Slack message to a channel specified by a Slack webhook variable.
 
-    A message is only sent if the SLACK_WEBHOOK environment variable is undefined.
+    A message is only sent if the SLACK_WEBHOOK environment variable is undefined,
+    and the environment is configured to log at this level.
     """
+    environment = config("ENVIRONMENT", default="local")
+
     if not (webhook := os.getenv(SLACK_WEBHOOK)):
         log.debug(
             f"{SLACK_WEBHOOK} variable not defined, skipping slack message: {text}"
         )
         return
+    # If no log level is configured in the environment, log everything by default.
+    os_level = os.getenv(LOG_LEVEL)
+    os_level = Level.VERBOSE if os_level is None else Level[os_level]
+    if not (os_level.value <= level.value):
```

Comment:
Would it be OK to invert this logic?
```suggestion
    if level.value < os_level.value:
```

Reply:
Totally, much easier to read 💯